.github/CODEOWNERS

```
@@ -1 +1 @@
-* @austinjpaul @mathcolo @friendchris @nathan-weinberg @skaplan-dev @devinmatte @PatrickCleary
+* @austinjpaul @skaplan-dev @devinmatte @PatrickCleary
```

We could do actual structured codeowners where people are tagged only on files they care about

```suggestion
* @austinjpaul @skaplan-dev @devinmatte @PatrickCleary
.github @nathan-weinberg
deploy.sh @mathcolo
```